### PR TITLE
Health EndPoint for FrontEnd

### PR DESCRIPTION
**Pull request by SyedSibtainRazvi**
# Related Issue

-https://github.com/code-mancers/gather-bot/issues/156

# functionalities Added or Changed

- Integrated API response in the front end for health

# Additional Info

- 

# Checklist

- [ ] Tests
- [x] Uploaded screenshots

# Screenshots
response fe
standup fe without health

### CODE REVIEW

File: client/src/apis/healthCheck.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import instance from ".";
+
+export const fetchHealth = async () => {
+  const response = await instance.get("/health");
```
Comment (SujayPrabhu96) [inline client/src/apis/healthCheck.ts:4]: Wrap it in try catch

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: client/src/apis/healthCheck.ts (hunk 1 above)
Comment (SyedSibtainRazvi) [reply client/src/apis/healthCheck.ts:4]: Doing a try catch in HealthCheck Component in UseEffect!

File: client/src/components/HealthCheck.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+const HealthCheck = () => {
+  const [health, setHealth] = useState("");
+
+  useEffect(() => {
+    const fetchHealthData = async () => {
+      const healthStatus = await fetchHealth();
+      setHealth(healthStatus);
+      console.log(healthStatus);
```
Comment (SujayPrabhu96) [inline client/src/components/HealthCheck.tsx:13]: remove

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: client/src/apis/healthCheck.ts (hunk 1 above)
Comment (SyedSibtainRazvi) [reply client/src/apis/healthCheck.ts]: Done

File: client/src/components/HealthCheck.tsx (hunk 2 above)
Comment (SyedSibtainRazvi) [reply client/src/components/HealthCheck.tsx:13]: Sure

File: client/src/components/HealthCheck.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { fetchHealth } from "../apis/healthCheck";
+import { useState, useEffect } from "react";
+
+
```
Comment (SujayPrabhu96) [inline client/src/components/HealthCheck.tsx:5]: keep only one line space

File: client/src/components/HealthCheck.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+const HealthCheck = () => {
+  const [health, setHealth] = useState("");
+
+  useEffect(() => {
+    const fetchHealthData = async () => {
+      const healthStatus = await fetchHealth();
+      setHealth(healthStatus);
+    };
+    fetchHealthData();
+  }, []);
```
Comment (SujayPrabhu96) [inline client/src/components/HealthCheck.tsx:15]: Add one line space after to differentiate the code blocks

File: client/src/components/HealthCheck.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  return (
```
Comment (SujayPrabhu96) [inline client/src/components/HealthCheck.tsx:16]: return `<p>{health}</p>`

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: client/src/components/HealthCheck.tsx (hunk 3 above)
Comment (SyedSibtainRazvi) [reply client/src/components/HealthCheck.tsx:5]: Done

File: client/src/components/HealthCheck.tsx (hunk 5 above)
Comment (SyedSibtainRazvi) [reply client/src/components/HealthCheck.tsx:16]: Done

File: client/src/components/HealthCheck.tsx (hunk 4 above)
Comment (SyedSibtainRazvi) [reply client/src/components/HealthCheck.tsx:15]: Done

**Pull request merged**